scripts/aws_clear_instance.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+echo "Removing application and dependencies"
+
+if [ -d "/home/notify-app/notifications-api" ]; then
+    # Remove and re-create the directory
+    rm -rf /home/notify-app/notifications-api
+    mkdir -vp /home/notify-app/notifications-api
+    # Remove installed py3 packages
+    pip3 freeze | xargs pip3 uninstall -y
+else
+    echo "Directory does not exist, something went wrong!"
```

Comment:
Looks like we're using that logic too in `scripts::register_with_elb.sh` on line 23 where we have `error_exit "Unable to get this instance's ID; cannot continue."`.